dataservice/util/data_gen/data.py

```diff
@@ -0,0 +1,121 @@
+"""
+This file contains numbers and choices for
+the entities
+"""
+import random
+import csv
+
+NUM_PARTICIPANTS = 10
+
+# NUM_DEMOGRAPHICS = 1
+
+NUM_DIAGNOSES = random.randint(0, 35)
```

Comment:
Setting these to a random number on import seems counter intuitive.
This way, all samples will have the same number of aliquots, but that number will change between runs.
Using a random number of children for each parent may be a better idea. Perhaps this number could be `MAX_DIAGNOSES` then

Reply:
Agreed. These numbers should either be constant totals for each entity type or maximums so that you can randomize the total (bounded by max) of that entity at runtime not during import